src/itally.cpp

```diff
@@ -0,0 +1,137 @@
+#include <tallies/box_position_filter.hpp>
+#include <tallies/general_tally.hpp>
+#include <tallies/itally.hpp>
+#include <tallies/legendre_fet.hpp>
+#include <tallies/regular_cartesian_mesh_filter.hpp>
+#include <tallies/tallies.hpp>
+#include <utils/output.hpp>
+
+double ITally::particle_base_score(const Particle& p, MaterialHelper& mat) {
+  double collision_score = 1.0 / (net_weight_);
```

Comment:
Divisions are expensive, and we need to do this one all the time. We should create a variable `invs_net_weight_ = 1. / net_weight_` which belongs to `ITally`. Then we can just multiply, which is much faster.

Reply:
Great Idea!